extensions/amp-accordion/amp-accordion.md

```diff
@@ -127,79 +133,68 @@ animation when the content is expanded and "roll up" animation when collapsed.
 
 [/example]
 
-[filter formats="websites"]
-
-### disable-session-states
-
-Include the `disable-session-states` attribute on `<amp-accordion>` to disable
-collapsed/expanded state preservation.
-
-[/filter] <!-- formats="websites" -->
-
 ### expanded
 
 Apply the `expanded` attribute to a nested `<section>` to expand that section when the page loads.
 
-### expand-single-section
-
-Apply the `expand-single-section` attribute to `amp-accordion` to specify that
-only one `<section>` can expand at a time. If the user clicks or taps on a
-collapsed `<section>`, any currently expanded `<section>` collapses.
+Bind the `expanded` attribute to a `<section>` to expand or collapse that
+section. An expanded section collapses if the expression evaluates as false. A
+collapsed section expands if the expression evaluates as anything other than
+false.
 
-[example preview="top-frame" playground="true"]
+[example preview="top-frame" playground="true" imports="amp-accordion:1.0"]
 
 ```html
-<amp-accordion expand-single-section>
-  <section>
+<amp-accordion>
+  <section
+    [expanded]="sectionOne"
+    on="expand:AMP.setState({sectionOne: true});collapse:AMP.setState({sectionOne: false})"
```

Comment:
/cc @caroqliu @sebastianbenz @CrystalOnScript  This is a good example of Bento-vs-AMP world. In Bento we recommend to call API, in AMP this is the right path. Not sure yet how to separate these.

Reply:
For now, I think it's best to have a separate example using the JS API in the bento section of the docs.

Reply:
To clarify, I should create another example in this doc?  Using a call to API?

Reply:
Yep. I'd have two sub-sections: AMP example and Bento example to call imperative APIs.

Reply:
The Bento example could be much smaller. E.g. "In Bento mode the same APIs can be called via `getApi()`, for instance:"

Reply:
Great!  This has been added here (at the end of the `Actions` section in the accordion documentation): https://github.com/ampproject/amphtml/pull/31920/files#diff-ab5b7d1f812d4c22e6a887e4324be32ec144b20fcf7075b971d968f65616dea5R260

Reply:
I've created a separate PR for further discussion on this!
https://github.com/ampproject/amphtml/pull/32052

@sebastianbenz @CrystalOnScript @dvoytenko